Ports/tinyscheme/patches/set-init-file.patch

```diff
@@ -0,0 +1,11 @@
+--- tinyscheme-1.42/scheme.c	2020-05-30 04:37:00.000000000 +0200
++++ tinyscheme-1.42-serenity/scheme.c	2021-02-17 13:34:19.957950383 +0100
+@@ -101,7 +101,7 @@
+ #endif
+
+ #ifndef InitFile
+-# define InitFile "init.scm"
++# define InitFile "/usr/local/include/tinyscheme/init.scm"
+ #endif
```

Comment:
Since this is inside an `#ifndef InitFile`, can't you simply pass `InitFile=/usr/local/include/tinyscheme/init.scm` to `make` instead of patching the source?

Reply:
Done. It simply didn't dawn on me that `-D` (re)defines constants in the source code itself.

Reply:
`-D` just defines macros before reading the source, but since the `#define InitFile` in the source is wrapped inside a `#ifndef InitFile` ("IF Not DEFined"), the `#define` in the source is never executed.